git-server/src/lib.rs

```diff
@@ -158,13 +227,29 @@ fn git(
 
     tracing::debug!("namespace: {}", namespace);
     tracing::debug!("path: {:?}", path);
+    tracing::debug!("username: {:?}", username);
+    tracing::debug!("method: {:?}", method.as_str());
+    tracing::debug!("remote: {:?}", remote.to_string());
 
     let mut cmd = Command::new("git");
 
     cmd.arg("http-backend");
 
+    // Set Authorized Keys for verifying GPG signatures against key IDs.
+    cmd.env("RADICLE_AUTHORIZED_KEYS", ctx.authorized_keys.join(","));
+    cmd.env(
+        "RADICLE_ALLOW_UNAUTHORIZED_KEYS",
+        ctx.allow_unauthorized_keys.to_string(),
+    );
```

Comment:
This makes me uneasy, can we actually not set the var at all if `allow_unauthorized_keys` is `false`? Seems safer to not expect that the value will be checked for true/false.

Reply:
5030fa7 updates.